lib/openstudio/analysis/version.rb

```diff
@@ -37,6 +37,6 @@ module OpenStudio
   module Analysis
     # format should be ^.*\-{1}[a-z]+[0-9]+
     # for example: -rc1, -beta6, -customusecase0
-    VERSION = '1.2.0'.freeze
+    VERSION = '1.3.0-0'.freeze
```

Comment:
release candidate?

```suggestion
    VERSION = '1.3.0-rc1'.freeze
```

Reply:
yeah, rc1 is prob better.  Tests using this branch in Server pass  https://github.com/NREL/OpenStudio-server/tree/analysis-gem-1-3-0-0